ddlite.py

```diff
@@ -78,6 +90,8 @@ class candidate_internal(object):
   See entity_internal and relation_internal for examples
   """
   def __init__(self, all_idxs, labels, sent, xt):
+    self.uid = "{}::{}::{}::{}".format(sent.doc_name, sent.sent_id,
```

Comment:
@xiaoling `doc_id` vs. `doc_name` is an unfortunate distinction due to "legacy" usage. `doc_id` is the integer index in the corpus, and `doc_name` is the actual file name. The idea is that if I add more documents to the corpus, the `doc_id`s might change but I may have information attached to specific candidates based on their metadata from the old run that I'd want to retrieve in the new run